plugins/database/mssql/mssql-database-plugin/main.go

```diff
@@ -0,0 +1,16 @@
+package main
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/hashicorp/vault/plugins/database/mssql"
+)
+
+func main() {
+	err := mssql.Run()
+	if err != nil {
+		fmt.Println(err)
```

Comment:
Does this end up going to Vault's log due to the way go-plugin forwards output? Either way is there any way this can be formatted "normally"?